.github/workflows/build-and-publish.yaml

```
@@ -0,0 +1,53 @@
+# Copyright The FMS HF Tuning Authors
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+name: Build and Publish FMS-hf-tuning Library
+
+on:
+    workflow_dispatch:
+        inputs:
+            release-version:
+              type: string
+              required: true
+              description: 'Version number (for example: 0.0.1)'
+
+jobs:
+  build:
+    runs-on: ubuntu-20.04
```

```suggestion
    runs-on: ubuntu-latest
```
unless you have special requirement must run on 20.04

---

I originally had it as ubuntu-latest but I was seeing warnings that the older nodejs was deprecated and we should move to 20... I can revert back to ubuntu-latest and let it ride for now, thanks!

---

switched back to `runs-on: ubuntu-latest`  thanks